fe/fe-core/src/main/java/com/starrocks/qe/LeaderOpExecutor.java

```diff
@@ -200,6 +200,8 @@ private void forward() throws Exception {
         if (setStmt != null) {
             params.setModified_variables_sql(AstToSQLBuilder.toSQL(setStmt));
         }
+
+        params.setWarehouse_id(ctx.getCurrentWarehouseId());
```

Comment:
should we add UT in `LeaderOpExecutorTest.java`

Reply:
OK. These code are synced from enterprise version, I had checked that the LeaderOpExecutorTest.java file are same. 

Whatever I will try to add some UT.

Reply:
abstracted a method, named createTMasterOpRequest, and added UT for it.